fluent/src/parser.js

```diff
@@ -131,32 +131,36 @@ class RuntimeParser {
       this._index++;
 
       this.skipInlineWS();
-      this.skipBlankLines();
-      this.skipInlineWS();
 
-      val = this.getPattern();
+      if (this._source[this._index] === '\n') {
+        this.skipBlankLines();
+        if (this._source[this._index] === ' ') {
+          this.skipInlineWS();
+          val = this.getPattern();
+        }
+      } else {
+        val = this.getPattern();
+      }
     } else {
-      this.skipWS();
+      this.skipInlineWS();
+      this.skipBlankLines();
     }
 
     ch = this._source[this._index];
 
-    if (ch === '\n') {
-      this._index++;
-      this.skipBlankLines();
+    if (ch === ' ') {
+      const lineStart = this._index;
       this.skipInlineWS();
+
       ch = this._source[this._index];
-    }
 
-    if (ch === '.') {
-      attrs = this.getAttributes();
-    }
+      this._index = lineStart;
 
-    if (ch === '#') {
-      if (attrs !== null) {
-        throw this.error('Tags cannot be added to a message with attributes.');
```

Comment:
Yeah. Since we don't support both, let's just consume one or the other and move on. This simplifies the code and we only pay with a different error message that gives less of a clue, which I think is ok, since our runtime errors are not meaningful at all anyway (since we optimize for perf).

Reply:
I agree we shouldn't care about the error itself, but the important thing here is to make the runtime parser yield the same Junks as the full syntax parser. Otherwise, as a developer you might write something which will work in testing, but will break in Pontoon / compare-locales etc.

Reply:
@zbraniecki made me realize that the `if-else` in lines 162-166 is enough to junkify an erroneous message here.